themes/red-theme/lms/templates/header.html

```diff
@@ -1,4 +1,5 @@
 ## mako
+<%page expression_filter="h"/>
 <%namespace name='static' file='static_content.html'/>
```

Comment:
This page has a `${site_status_msg}` that you should confirm doesn't support HTML, or this will cause unwanted escaping. If it does allow HTML, you'll want to ensure it is safe.